docs/kyma/03-05-resource-quotas.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Resource quotas
+type: Details
+---
+
+[Resource quotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/) are a convenient way to manage the consumption of resources in a Kyma cluster. You can easily set resource quotas for every Namespace you create through the Console UI.
+
+When you click **Create Namespace**, you can define:
+  - Total Memory Quotas, which limit the overall memory consumption by the Namespace by creating a ResourceQuota object.
```

Comment:
```suggestion
  - Total Memory Quotas, which limit the overall Namespace memory consumption by creating ResourceQuota objects.
```

Reply:
Resource quotas or ResourceQuotas? Whichever you choose, be consistent.

Reply:
Resource quotas are set through a ResourceQuota Kubernetes object.

Reply:
Oh, and there's a single ResourceQuota object which limits how much resources the Namespace can consume.